tests/test_ddk.py

```diff
@@ -105,26 +123,40 @@ def test_sini_from_value(self):
             self.modelJ1713.validate()
 
     def test_sini_from_par(self):
-        temp_par_str = """PSR  J1713+0747
-               LAMBDA 256.66  1 0.001
-               BETA 30.70036  1 0.001
-               PMLAMBDA 5.2671  1  0.0021
-               PMBETA  -3.4428  1  0.0043
-               PX  0.8211  1  0.0258
-               F0  218.81  1  0.01
-               PEPOCH  55391.0
-               BINARY  DDK
-               A1 32.34  1  0.001
-               E  0.074  1  0.001
-               T0 55388.836  1  0.0002
-               PB 67.825129  1  0.0001
-               OM 176.19845  1  0.0013
-               M2  0.283395  1  0.0104
-               KOM   83.100  1  1.800
-               SINI  0.8     1  0.562
-               K96  1"""
+        test_par_str = self.temp_par_str + "\n SINI  0.8     1  0.562"
+        with pytest.raises(ValueError):
+            mb.get_model(StringIO(test_par_str))
+
+    def test_stand_alone_model_params(self):
```

Comment:
It is more like verify_stand_alone_model_parameters_are_updated_from_pint_facing_object. I think this can be a generic function. Should I make a test_utils.py file to host this type of function?

Reply:
If the tests that use such a function all live in the same file, support functions can just live in that file. Support functions that are needed more generally, I'm not sure what the best approach is - if the support functions are potentially useful for PINT users (rather than just PINT developers) they could just go in `pint.utils`; if they're really only useful in the context of PINT's test suite, there's some standard way to do that - maybe just a module in `tests/` that doesn't start with `test_` so it doesn't get picked up as containing test cases? Other test files could just import it then.

Reply:
I made a tests/utils.py for hosting this type of function.